docs/sphinx/manual/launching_and_configuring_hpx_applications.rst

```diff
@@ -465,6 +465,7 @@ The ``hpx.parcel`` configuration section
     max_outbound_message_size = ${HPX_PARCEL_MAX_OUTBOUND_MESSAGE_SIZE:<hpx_parcel_max_outbound_message_size>}
     array_optimization = ${HPX_PARCEL_ARRAY_OPTIMIZATION:1}
     zero_copy_optimization = ${HPX_PARCEL_ZERO_COPY_OPTIMIZATION:$[hpx.parcel.array_optimization]}
+    zero_copy_receive_optimization = ${HPX_PARCEL_ZERO_COPY_RECEIVE_OPTIMIZATION:$[hpx.parcel.array_optimization]}
```

Comment:
Should the default value be `$[hpx.parcel.array_optimization]`? I think it can just be `1`?

Reply:
We can't logically support zero-copy networking with array optimization is off.  However, I think the default should be `$[hpx.parcel.zero_copy_optimization]`. I'll make that change.